SIL.WritingSystems/LdmlDataMapper.cs

```diff
@@ -819,6 +827,7 @@ public void Write(string filePath, WritingSystemDefinition ws, Stream oldFile)
 				using (var writer = XmlWriter.Create(filePath, writerSettings))
 				{
 					WriteLdml(writer, element, ws);
+					writer.Flush();
```

Comment:
I don't think this Flush() call should be necessary. Doesn't Close() flush the buffers? In fact, the Close() shouldn't even be necessary since the XmlWriter is wrapped in a using statement.